src/alfasim_sdk/result_reader/aggregator.py

```diff
@@ -105,57 +121,131 @@ class ResultsNeedFullReloadError(RuntimeError):
     """
 
 
+@attr.s(slots=True, hash=False)
+class BaseUQMetaData:
+    """
+    Base class for UQ metadata analysis.
+    :ivar property_id:
+        The property name (holdup, total mass flow rate).
```

Comment:
```suggestion
    """
    Base class for UQ metadata analysis.

    :ivar property_id:
        The property name (holdup, total mass flow rate).
```
This is not the first reminder about that empty line in the docs to isolate the sort descripton from the body of the documentation.

Reply:
I know that. I'm paying attention on this, but sometimes, some cases passes.